files/aws_ips_autodiscover

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+
+function main {
+  INSTANCE_REGION=$(curl -s -f http://169.254.169.254/latest/meta-data/placement/availability-zone | sed '$s/.$//')
+  if [[ -z ${INSTANCE_REGION} ]]; then
+    exit 1
+  fi
+
+  local IPS
+  IPS=( $( aws ec2 describe-instances \
+    --region ${INSTANCE_REGION} \
+    --filters Name=tag:Env,Values=prd Name=tag:Role,Values=consul_server Name=instance-state-name,Values=running \
```

Comment:
this should be tempalte not a file using parameter `aws_ips_autodiscover.lookup_filter` - I will fix it, no worries.